### ISSUE TITLE
In "dropdown or text"-parameter: highlight that that the open-text field is created automatically in addition to the drop down choices.

### ISSUE THREAD

**Issue by Pedrotheplant** [nikobojs/manuscrape_electron#40]

**Comment by nikobojs** [comment]
So the popup/dialog/modal where you pick the options for the dropdown/radio/etc. We need an explainer text inside that dialog, if user picked the "dropdown or text" type. Is that correct?

**Comment by Pedrotheplant** [comment]
Yes, e.g., when typing in the options there is already a pre-set option called "open text field". So it appears next to all the other options the project owner creates. Also, is it possible to for the project owner to name the open text field? E.g., "'Other:' [open text field]"?

**Comment by nikobojs** [comment]
Will take a look and see what makes sense.

Right now the dialog is the same for configuring all multiple choice types (radios+dropdown+dropdown/text). That generic behavior needs to change, to support different dialog config options for each type.

Is it correctly understood that the "naming" of the open text field, _only_ affects how it looks, and not the excel export / parameters list ?

**Comment by Pedrotheplant** [comment]
Yes that is correct
